_posts/2020-10-21-announcing-presto-conference-tokyo-2020.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title:  "Announcing Presto Conference Tokyo 2020"
+author: Yuya Ebihara, LINE
+excerpt_separator: <!--more-->
+---
+
+Last year, [Presto Conference Tokyo 2019]({% post_url 2019-07-11-report-for-presto-conference-tokyo %}) 
+was held in Japan with Martin Traverso, Dain Sundstrom and David Philips, 
```

Comment:
Typo: Phillips

Reply:
Fixed. Sorry, I copied from this event page. Let me ask Toru (Treasure Data) these page.
https://techplay.jp/event/733772
https://techplay.jp/event/795265